est/001.est

```diff
@@ -0,0 +1,20 @@
+date: 24-02-2015
+author: Carlos Miranda
+method: champions.pert
+scope:
+  1: Website UI design
+  2: API design 
+  3: User addition, modification, and deletion of repos
+  4: Automatic deployment of user repos
+  5: Billing of resource usage
+  6: Resource scaling functionality
+  7: Integration testing
+champions:
+  2:
+    worst-case: 40
```

Comment:
I applied the techniques described in https://github.com/teamed/est. I think every estimator can have very different numbers, which is integral to the methodology itself. :)